e2e/tests/sideMenuLinksCheck.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from '../src/FixtureConfigs';
+import { elementsAreVisible, getAllLinks, verifyLinksResponse } from '../src/commonFunctions';
+
+test.describe('Side Menu Navigation', () => {
+    test('Verify side menu functionality and link navigation', {
+        tag: '@regression'
+    }, async ({ homePage }) => {
+        await homePage.openAndVerify();
+
+        await homePage.header.clickOnBurgerMenu();
+        await homePage.sideMenu.sidebar.isVisible();
+        await elementsAreVisible(homePage.sideMenu.allLinks);
+
+        const linksUrls = await getAllLinks(homePage.sideMenu.sidebar, await homePage.getPageUrl());
+        await verifyLinksResponse(linksUrls);
+    });
+});
```

Comment:
<img width="218" alt="image" src="https://github.com/user-attachments/assets/b169aa4d-4edf-46d0-9a22-d3fa62263095" />
As for the names, you add the word Checks at the end. Why didn't you add this word to the footer test, it would be cool if our tests had a similar naming pattern